### ISSUE TITLE
OkHttp error

### ISSUE THREAD

**Issue by sohayb** [auth0/Lock.Android#457]
Hello, I'm using `Lock 2.8.2`
Whenever I want to launch the `Lock`'s authentication `activity`, I'm getting this screen.
<img src="https://user-images.githubusercontent.com/2195533/37551769-9e830364-29ea-11e8-90b8-dfc63694462d.png" width="320" height="480">
I debugged the issue and it turned out that there's an error returned in 

```java
com.auth0.android.lock.internal.configuration.ApplicationFetcher#makeApplicationRequest`
```
 and `onFailure` is invoked. with this error: `Expected ':status' header not present`.

After checking online, what I found is that this is a client-server issue that is either resolved by upgrading `OkHttp` (I found that a pretty old version is being used) or adding some other configuration to it, or it was caused by a server issue. In both cases, I cannot control these.
Please note that calling the url from `Chrome` worked (both on mac and in emulator) and returned valid JSON. Also note that this is a new issue, I wasn't facing this issue a couple of days ago, I was able to login normally, also, I'm still able to login normally from iOS.

I'm using  Nexus 5X emulator with API 27 (Google Play version)

**Comment by lbalmaceda** [comment]
Ahhgg what an ugly thing.. 😞 I could successfully confirm the issue. ⚡️  
Note that 2.7.5 is the latest version for that major, so updating it would mean moving to OkHttp3 and that is a breaking change as the `auth0.android` Requests classes' methods would need to be changed to include `Call` AFAIK by checking the open okttp3 PRs.
This seems to be protocol related and enforcing Http1.1 on the client fixes it. I'll add a patch ASAP.

**Comment by sohayb** [comment]
Thanks for the fix! Since you're not releasing `2.8.3` now, for anyone who's having the same issue, just exclude the old `auth0` module and include the new one:
```groovy
    implementation('com.auth0.android:lock:2.8.2') {
        exclude group: 'com.auth0.android', module: 'auth0'
    }
    implementation 'com.auth0.android:auth0:1.12.2'
```

**Comment by lbalmaceda** [comment]
It was released on bintray [~12hs ago](https://bintray.com/auth0/android/lock/2.8.3).  Maven central usually takes a few more hours but should be there too. 
Cheers
